lib/little-loader.js

```diff
@@ -126,4 +135,19 @@
       _addScript(script);
     }
   };
+
+  // UMD wrapper.
+  /*global define:false*/
+  if (typeof exports === "object" && typeof module === "object") {
```

Comment:
Then I'd have to have _something_ left in here for `var _lload`, probably window and would have to process this out. Might be nice for you to write a ticket of "split different output builds to `dist/`" in the future though...